### PR TITLE
Issue 143

### PR DESCRIPTION
**Pull request by gaetan-h**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
#### Checkmarx scan completed with the following findings
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[88](https://github.com/cognitedata/cognite-replicator/blob/issue_143/cognite/replicator/__main__.py#L88) [77](https://github.com/cognitedata/cognite-replicator/blob/issue_143/cognite/replicator/__main__.py#L77) |Medium|Path_Traversal|cognite/replicator/__main__.py|[Checkmarx](https://cognite.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1023905&projectid=145&pathid=2)

File: cognite/replicator/__main__.py [hunk 1]
```diff
@@ -25,6 +25,9 @@
 
 from . import assets, configure_logger, datapoints, events, files, raw, time_series
 
+# import assets, datapoints, events, files, raw, time_series
```
Comment (magneei) [inline cognite/replicator/__main__.py:28]: Why are these comments added to the file?

File: cognite/replicator/__main__.py [hunk 2]
```diff
@@ -25,6 +25,9 @@
...
+# from __init__ import configure_logger
```
Comment (magneei) [inline cognite/replicator/__main__.py:29]: Should this comment be here?

File: cognite/replicator/replication.py [hunk 3]
```diff
@@ -146,6 +146,55 @@ def new_metadata(
...
+
+    for key in exclude_fields:
+        if key == "name":
+            dst_obj.name = dst_obj_dump["name"]
+        elif key == "description":
+            dst_obj.description = dst_obj_dump["description"]
+        elif key == "metadata":
+            replicator_metadata = {}
+
+            for replicator_metadata_field in replicator_metadata_fields:
```
Comment (magneei) [inline cognite/replicator/replication.py:183]: I'm not sure, but why do you need to loop through all fields? Can't you just set dst_obj.metadata = dst_obj_dump["metadata"] when you want to skip metadata sync?

File: cognite/replicator/time_series.py [hunk 4]
```diff
@@ -10,6 +10,8 @@
 
 from . import replication
 
+# import replication
```
Comment (magneei) [inline cognite/replicator/time_series.py:13]: Remove the comment?

**Review by magneei — COMMENTED** [review]
Added a few comments which should be addressed before approval

File: cognite/replicator/__main__.py (hunk 1 above)
Comment (gaetan-h) [reply cognite/replicator/__main__.py:28]: Good question, I think he has changed the way of the import, changed it

File: cognite/replicator/__main__.py (hunk 2 above)
Comment (gaetan-h) [reply cognite/replicator/__main__.py:29]: nope

File: cognite/replicator/replication.py (hunk 3 above)
Comment (gaetan-h) [reply cognite/replicator/replication.py:183]: I think it is looping through those fields and adding them anyway

    replicator_metadata_fields = ["_replicatedSource", "_replicatedTime", "_replicatedInternalId"]

It is ok no?

Comment (gaetan-h) [reply cognite/replicator/replication.py:183]: or we dont want any metadata at all, i think the replicator relies on those fields to be there

File: cognite/replicator/time_series.py (hunk 4 above)
Comment (gaetan-h) [reply cognite/replicator/time_series.py:13]: gone

**Comment by gaetan-h** [comment]
Thanks @magneei I have updated the code. 
Let me know if ok
If not, I will check with Jorge

**Review by magneei — APPROVED** [review]
LGTM

**Pull request merged**